dataretrieval/waterdata/api.py

```diff
@@ -868,93 +843,85 @@ def get_latest_continuous(
-    args = { 
-        k: v for k, v in locals().items()
+    args = {
+        k: v
+        for k, v in locals().items()
         if k not in {"service", "output_id"} and v is not None
     }
 
-    return waterdata_helpers.get_ogc_data(args, output_id, service)
+    return get_ogc_data(args, output_id, service)
+
 
 def get_field_measurements(
-        monitoring_location_id: Optional[Union[str, List[str]]] = None,
-        parameter_code: Optional[Union[str, List[str]]] = None,
-        observing_procedure_code: Optional[Union[str, List[str]]] = None,
-        properties: Optional[List[str]] = None,
-        field_visit_id: Optional[Union[str, List[str]]] = None,
-        approval_status: Optional[Union[str, List[str]]] = None,
-        unit_of_measure: Optional[Union[str, List[str]]] = None,
-        qualifier: Optional[Union[str, List[str]]] = None,
-        value: Optional[Union[str, List[str]]] = None,
-        last_modified: Optional[Union[str, List[str]]] = None,
-        observing_procedure: Optional[Union[str, List[str]]] = None,
-        vertical_datum: Optional[Union[str, List[str]]] = None,
-        measuring_agency: Optional[Union[str, List[str]]] = None,
-        skipGeometry: Optional[bool] = None,
-        time: Optional[Union[str, List[str]]] = None,
-        bbox: Optional[List[float]] = None,
-        limit: Optional[int] = None,
-        max_results: Optional[int] = None,
-        convertType: bool = True
-        ) -> pd.DataFrame:
-    """Field measurements are physically measured values collected during
-    a visit to the monitoring location. Field measurements consist of
-    measurements of gage height and discharge, and readings of groundwater
-    levels, and are primarily used as calibration readings for the automated
-    sensors collecting continuous data. They are collected at a low frequency,
-    and delivery of the data in WDFN may be delayed due to data processing
-    time.
+    monitoring_location_id: Optional[Union[str, List[str]]] = None,
+    parameter_code: Optional[Union[str, List[str]]] = None,
+    observing_procedure_code: Optional[Union[str, List[str]]] = None,
+    properties: Optional[List[str]] = None,
+    field_visit_id: Optional[Union[str, List[str]]] = None,
+    approval_status: Optional[Union[str, List[str]]] = None,
+    unit_of_measure: Optional[Union[str, List[str]]] = None,
+    qualifier: Optional[Union[str, List[str]]] = None,
+    value: Optional[Union[str, List[str]]] = None,
+    last_modified: Optional[Union[str, List[str]]] = None,
+    observing_procedure: Optional[Union[str, List[str]]] = None,
+    vertical_datum: Optional[Union[str, List[str]]] = None,
+    measuring_agency: Optional[Union[str, List[str]]] = None,
+    skip_geometry: Optional[bool] = None,
+    time: Optional[Union[str, List[str]]] = None,
+    bbox: Optional[List[float]] = None,
+    limit: Optional[int] = None,
+    max_results: Optional[int] = None,
+    convert_type: bool = True,
+) -> pd.DataFrame:
```

Comment:
```suggestion
) -> Tuple[pd.DataFrame, BaseMetadata]:
```
I find these Tuples annoying to work with, but if it's standard for web service requests, then I can live with it.